frontend_tests/node_tests/settings_bots.js

```diff
@@ -65,6 +65,22 @@ var settings_bots = require("js/settings_bots.js");
 
     $("#create_bot_form").validate = function () {};
 
+    $('#create_bot_type').on = function (action, f) {
+        if (action === 'change') {
+            var create_payload_url = $('#create_payload_url');
+            var OUTGOING_WEBHOOK_BOT_TYPE = '3';
+            var GENERIC_BOT_TYPE = '1';
+
+            $('#create_bot_type :selected').val(OUTGOING_WEBHOOK_BOT_TYPE);
+            f.apply();
+            assert(create_payload_url.hasClass('required'));
```

Comment:
You can `assert` if `create_payload_url` is shown too. I think it's `create_payload_url.visible()`.